### ISSUE TITLE
Conditional field layout within blocks

### ISSUE THREAD

**Issue by jmchat** [verbb/vizy#64]
**What are you trying to do?**
A description of what you want to happen.
I wan't to make blocks easier and more flexible to be used by content editors. So if a certain setting is checked inside a block show fields or hide other fields which are irrelevant.

**What's your proposed solution?**
A description of how you think this could be solved, including any alternatives that you considered.

Just look at https://plugins.craftcms.com/reasons :-) and get it to work withn the Vizy blocks

**Additional context**

**Comment by engram-design** [comment]
Yep, we're going to wait for Craft 4 which introduces field conditionals, to piggyback off that functionality!

**Comment by jmchat** [comment]
https://github.com/craftcms/cms/pull/10300 Go for it 👼 😄

**Comment by JshGrn** [comment]
Is this still on the road-map? Hopefully simple to piggyback :)

**Comment by engram-design** [comment]
It's much more complicated than that, unfortunately. We'd need to build in conditionals completely from scratch, as the Vizy field is using Vue, and the conditions use jQuery and htmx - they aren't really compatible.

So this will have to go on the long-term roadmap.

**Comment by JshGrn** [comment]
Completely understandable, they do not mix.

**Comment by JayBox325** [comment]
Hi @engram-design, is there any update on the plans to implement this at all or is it still on the back-burner?

**Comment by engram-design** [comment]
No immediate plans, sorry. We're very much in the camp of we'd need to build it all ourselves, which is a bit of a task.

We're working on some larger updates to Vizy, so we may very well try and bundle this in with that, we'll have to see.

**Comment by JayBox325** [comment]
Ok mate, thanks for the update.
